lib/agenda.js

```diff
@@ -439,8 +488,14 @@ Agenda.prototype.now = function(name, data, cb) {
  *  @returns {undefined}
  */
 Agenda.prototype.cancel = function(query, cb) {
+  debug('attempting to cancel all Agenda jobs', query);
   this._collection.deleteMany( query, function( error, result ){
     if (cb) {
+      if (error) {
+        debug('error trying to delete jobs from MongoDB');
+      } else {
+        debug('job cancellation was successful!');
```

Comment:
This could be just "_job cancelled_"?

I kinda like idea adding job name or id there but not sure if that would actually be useful or just add noise.